### DISCUSSION TITLE
About verifying my email

### DISCUSSION THREAD

**Discussion by Ogannesson** [community/community#17716] [GitHub Education]
I added a second email to my account, which is an educational email from my university, but I can't receive the verification code that github sends me

**Comment by anonys6** [comment]
Try to login your university email in portal.office.com
There use outlook to see all the emails

**Comment by mlarasusan** [comment]
Hello [Unbiseptium](https://github.com/Unbiseptium). Did you [verify it](https://help.github.com/en/articles/verifying-your-email-address)? Unverified email addresses will not appear. Once verified, refresh the application page for your email to appear. Thank you :]

**Reply by Ogannesson** [reply]
After contacting github support, it was determined that the verification email was rejected by my email service provider. After contacting the email admin, I have successfully verified it.

**Reply by mlarasusan** [reply]
Awesome, thanks for sharing it with us.
